lib/classes/ConfigSchemaHandler/normalizeAjvErrors.js

```diff
@@ -201,15 +196,15 @@ const filterIrrelevantAnyOfErrors = resultErrorsSet => {
 const normalizeDataPath = dataPath => {
   if (!dataPath) return 'root';
 
-  // This regex helps replace functions['someFunc'].foo with functions.someFunc.foo
-  return `'${dataPath.slice(1).replace(dataPathPropertyBracketsPattern, '.$1')}'`;
```

Comment:
I understand that AJV with v7 no longer puts such literl property names into brackets, so this optimization is no longer needed (?)

Reply:
@medikoo `functions['someFunc'].foo` will now be referenced by ajv as `functions/someFunc/foo`, so this optimization is indeed no longer needed. I kept it for the leading `/` removal. If you'd rather stick with `ajv` path normalizer, I can remove it altogether

Reply:
> @medikoo functions['someFunc'].foo will now be referenced by ajv as functions/someFunc/foo, so this optimization is indeed no longer needed

Great to hear.

Still I wonder whether we should not stick to `.` as delimiter in our schema error report messages. It seems more natural delimiter. What do you think?

If we agree on that, then I believe this function (of which purpose is to normalize property path for the error message) should be updated to replace `/` with `.`

Reply:
Agreed. I updated the PR accordingly :)